<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1728</width>
    <height>1278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TICKETING</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="SWMain">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">border-radius:0px;</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QGroupBox" name="GBMain">
             <property name="styleSheet">
              <string notr="true">background:transparent;</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="minimumSize">
                 <size>
                  <width>700</width>
                  <height>600</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">border-image: url(:/Resources/imgs/taco.png) 0 0 0 0 stretch stretch;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lblTacontento">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>600</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Montserrat</family>
                  <pointsize>35</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background:transparent;</string>
                </property>
                <property name="text">
                 <string>Tacontento</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="lblFeliz">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Montserrat</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">background:transparent;</string>
                </property>
                <property name="text">
                 <string>El que te hace feliz</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>950</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background:url(:Resources/imgs/2.jpeg);</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="maximumSize">
              <size>
               <width>700</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background:transparent;
border-radius:0px;</string>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="lblOrdena">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>400</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Montserrat</family>
                  <pointsize>40</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>¡Ordena ahora!</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QPushButton" name="empezarPB">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(181, 181, 181);
selection-background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
                </property>
                <property name="text">
                 <string>¡Empieza a ordenar!</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="lblNombre">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>400</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Montserrat</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Ingresa su nombre</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="lblMesa">
                <property name="maximumSize">
                 <size>
                  <width>600</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Montserrat</family>
                  <pointsize>20</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Ingresa el código de la mesa</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLineEdit" name="LENombre">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="placeholderText">
                 <string>Nombre</string>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLineEdit" name="LEMesa">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>50</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
                </property>
                <property name="placeholderText">
                 <string>Mesa</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_7">
        <item row="0" column="0">
         <widget class="QGroupBox" name="GBLoginMeseros">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>300</height>
           </size>
          </property>
          <property name="title">
           <string>Login</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="lblIDMeseros">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="LEIDMeseros">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>30</height>
              </size>
             </property>
             <property name="placeholderText">
              <string>ID</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblPasswordMeseros">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Password:</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLineEdit" name="LEPasswordMeseros">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>30</height>
              </size>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QPushButton" name="PBLoginMeseros">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1728</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
